src/main/java/htsjdk/samtools/util/htsget/HtsgetErrorResponse.java

```diff
@@ -0,0 +1,35 @@
+package htsjdk.samtools.util.htsget;
+
+
+/**
+ * Class allowing deserialization from json htsget error response
+ */
+public class HtsgetErrorResponse {
```

Comment:
Maybe put an example of what the json scheme looks like in a comment here?  Or a link to the spec.